src/websocket.ts

```diff
@@ -3,8 +3,11 @@ import Queue from "./queue";
 import act from "./act-compat";
 
 const identity = (x: string) => x;
+
 interface WSOptions {
   jsonProtocol?: boolean;
+  verifyClient?: () => boolean;
+  selectProtocol?: (protocols: string[]) => string;
```

Comment:
Can you import `ServerOptions` from mock-socket and just have this interface extend it to add the `jsonProtocol` option? 
https://github.com/thoov/mock-socket/blob/master/index.d.ts#L88